addons/l10n_in_edi/tests/test_edi_json.py

```diff
@@ -58,7 +58,7 @@ def setUpClass(cls, chart_template_ref="l10n_in.indian_chart_template_standard")
         cls.invoice_zero_qty.action_post()
 
     def test_edi_json(self):
-        # json_value = self.env["account.edi.format"]._l10n_in_edi_generate_invoice_json(self.invoice)
```

Comment:
@jva-odoo Why was this commented in the first place?

Reply:
it's commented from this PR https://github.com/odoo/odoo/pull/99401
And I don't know why it's commented!

Reply:
yes